1.0/tools/tests.md

```diff
@@ -30,6 +30,12 @@ popular third-party tools, including:
 This guide refers to Web Component Tester as `wct`, which is its command 
 line interface.
 
+Note: You can also use [polyserve](https://github.com/PolymerLabs/polyserve)
+to run your test suite in the browser. This allows you to use all browser
```

Comment:
"This allows you to use browser features like dev tools and HTML inspection to debug your unit tests."